servicetalk-http-api/src/testFixtures/java/io/servicetalk/http/api/DelegatingHttpExecutionStrategy.java

```diff
@@ -33,6 +33,21 @@ public DelegatingHttpExecutionStrategy(final HttpExecutionStrategy delegate) {
         this.delegate = requireNonNull(delegate);
     }
 
+    @Override
+    public boolean hasOffloads() {
+        return delegate.hasOffloads();
+    }
+
+    @Override
+    public boolean isRequestResponseOffloaded() {
+        return HttpExecutionStrategy.super.isRequestResponseOffloaded();
```

Comment:
Should this method call `delegate`? If not, please leave a comment explaining why

Reply:
Appears to be a bad default method implementation, fixed.